src/services/getOneRestaurant.js

```diff
@@ -0,0 +1,13 @@
+import { getOneRecord } from '../db/getOneRecord';
+import { createRestaurantPK } from '../libs/createRestaurantPK';
+
+export const getOneRestaurant = async (req, res, next) => {
+    const { name, street, postalCode } = req.query;
+    const pk = createRestaurantPK(name, street, postalCode);
```

Comment:
`createRestaurantPK` should probably take one object rather than 3 separate arguments. it gets confusing later on as you need to remember the order of arguments.

```
createRestaurantPK({ name, street, postalCode });
```